wordpress-mobile/WordPress-iOS
# Jetpack Landing Screen - Change gradient and button colors

Ref #19357

This PR addresses part of the #19357 issue, specifically changes the button colors and the gradient.

dark mode | light mode
-- | --
<img width=300 src=https://user-images.githubusercontent.com/34376330/194176262-9cf87d88-165f-4620-b711-9d52b2211f6c.png> | <img width=300 src=https://user-images.githubusercontent.com/34376330/194176310-350e4559-1128-4e7b-a32b-f3fe2102556f.png>



To test:

- checkout this branch and set the `newLandingScreen` feature flag to true
- Build the `Jetpack` target and logout if needed
- Make sure the gradient and button colors match the requirements (see linked issue and screenshots)


## Regression Notes
1. Potential unintended areas of impact
none

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
